cmp/ag-grid/AgGridModel.js

```diff
@@ -409,7 +416,7 @@ export class AgGridModel {
     }
 
     getGroupNodePath(node) {
-        throwIf(!node.group, 'Cannot get the path for a non-group node!');
+        throwIf(!node.allChildrenCount, 'Cannot get the path for a non-group node!');
```

Comment:
I see that the two callers to this method both appear to have already run this check.  Wondering what the value is here - especially as we've changed the check to look for children, and I could imagine some valid future usage hitting would-be-group-nodes that happen to not have children due to a filter or similar.  

Would the code below throw? I like these internal assertions where they help catch errors and make dev-time mistakes more clear with helpful exceptions, but not sure that applies here.